packages/coordinator/ts/sessionKeys/README.md

```diff
@@ -0,0 +1,9 @@
+# Session Keys module
+
+This module is responsible for session keys management. These keys (private are stored in the service, public are sent to the client) are used to generate the ZeroDev KernelClient. This way we are not storing any master keypair in the server that could be stolen. The point of these sessions keys were that they would be short-lived (~2min). All its endpoints are protected by the `AccountSignatureGuard` middleware, which checks the signature of the request against the `COORDINATOR_ADDRESSES` environment variable.
```

Comment:
I'm thinking that eventually we could even remove the account signature guard from certain endpoints like deployment as with the session key flow we use the initiator account not the coordinator one (we'd need to properly manage local storage of deployed addresses though as calls by others could cause deletion of the json file) but for now this works

Reply:
We could save multiple MACI instances (and their polls) in the coordinator but I will prefer to keep it as it is: one coordinator server per MACI instance. By keeping it separate we avoid single-point-of-failure attacks (e.g. imagine attacking only one server to DoS the deployment of polls for all DAOs. It would be harder to attack N servers).